src/ad-engine/services/slot-service.ts

```diff
@@ -113,6 +113,30 @@ class SlotService {
 		return slotByPos;
 	}
 
+	/**
+	 * Get slot id by its name
+	 */
+	getId(id: string): string {
```

Comment:
```suggestion
	getSlotId(id: string): string {
```
and jsdoc seems not needed

Reply:
I removed this method completely for now. It's not used and it won't be until `adengadinfo` refactor.

Reply:
No, that's not the part I removed :D Renamed!